core/arch/interp.c

```diff
@@ -4656,7 +4656,7 @@ static void
 build_native_exec_bb(dcontext_t *dcontext, build_bb_t *bb)
 {
     instr_t *in;
-    opnd_t jmp_tgt;
+    opnd_t jmp_tgt IF_AARCH64(UNUSED);
```

Comment:
Isn't this used on L4748 below for AArch64?

```
jmp_tgt = opnd_create_pc(bb->start_pc);
```

Reply:
It is only set there: never read after that for a64.  This clang 14 is very aggressive at these warnings: as you see it found tons of such vars.